### PR TITLE
Adjusts Gamemode Numbers

### PR DESCRIPTION
**Pull request by nachomeep**
For real this time. Sorry, Karol, the GitHub is dirty.

### CODE REVIEW

File: code/game/gamemodes/mixed/traitorrenegade.dm [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	round_description = "Subversive elements have invaded the station, and certain individuals are feeling uncertain about their safety."
 	extended_round_description = "Traitors and renegades spawn during this round."
 	config_tag = "traitorren"
-	required_players = 11			//I don't think we can have it lower and not need an ERT every round.
-	required_players_secret = 11	//I don't think we can have it lower and not need an ERT every round.
+	required_players = 4			//I don't think we can have it lower and not need an ERT every round.
```
Comment (Karolis2011) [inline code/game/gamemodes/mixed/traitorrenegade.dm]: Do you realize that setting this to 4 makes it spawn all players as an tags? I recommend to revert some numbers back.

File: code/game/gamemodes/mixed/traitorling.dm [hunk 2]
```diff
@@ -3,8 +3,8 @@
 	round_description = "There are traitors and alien changelings on the station. Do not let the changelings succeed!"
 	extended_round_description = "Traitors and changelings both spawn during this mode."
 	config_tag = "traitorling"
-	required_players = 10
-	required_players_secret = 10
+	required_players = 5
```
Comment (Karolis2011) [inline code/game/gamemodes/mixed/traitorling.dm]: Same as with traitorren

**Comment by nachomeep** [comment]
...right. Oops.

**Comment by nachomeep** [comment]
But renegades don't count as antags, by the way.

**Pull request merged**